src/x-platform/index.js

```diff
@@ -1,26 +1,107 @@
+import 'html5-desktop-notifications'
+import {notificationsConfig} from './constants'
+import noop from 'lodash/utility/noop'
+
+
+// Electron brings own require to window
+let electron
+let remote
+if (window.require) {
+  electron = require('electron')
+  remote = require('remote')
+}
+
+const macGap = window.MacGap
```

Comment:
please create a separate issue to remove macgap at some point ... we can't keep it forever ...